tests/cli_tests.py

```diff
@@ -66,34 +66,25 @@ class RDMToControlledListsETLTests(TestCase):
     @classmethod
     def setUpTestData(cls):
 
-        skos = SKOSReader()
-        rdf = skos.read_file(
-            os.path.join(PROJECT_TEST_ROOT, "data", "concept_label_test_collection.xml")
-        )
-        ret = skos.save_concepts_from_skos(rdf)
-
-        client = Client()
-        client.login(username="admin", password="admin")
-        response = client.get(
-            reverse(
-                "make_collection",
-                kwargs={"conceptid": "7c90899a-dbe9-4574-9175-e69481a80b3c"},
-            )
+        management.call_command(
+            "loaddata",
+            "tests/data/polyhierarchical_collections.json",
+            format="json",
```

Comment:
It's even easier than this, you can just use the `fixtures` class attribute. Let's use the standard `fixtures` location, otherwise we need to set a [FIXTURE_DIRS](https://docs.djangoproject.com/en/5.1/ref/settings/#fixture-dirs)